db/dohast.c

```diff
@@ -427,6 +428,24 @@ static dohsql_node_t *gen_union(Vdbe *v, Select *p, int span)
     return node;
 }
 
+static int skip_tables(Select *p)
```

Comment:
[Optional] s/skip_tables/skip_system_tables ?

Reply:
This is a callback to skip arbitrary tables, not only internal ones.